integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -968,7 +969,8 @@ public void testCreateNewCluster() {
   /**
    * Modify the domain scope property
    * From: "image: container-registry.oracle.com/middleware/weblogic:12.2.1.4" to
-   * To: "image: container-registry.oracle.com/middleware/weblogic:14.1.1.0-11"
+   * To: "image: container-registry.oracle.com/middleware/weblogic:DateAndTimeStamp"
+   * e.g, "image:container-registry.oracle.com/middleware/weblogic:2021-07-08-162571383699"
    * Verify all the pods are restarted and back to ready state
    * Verify the admin server is accessible and cluster members are healthy
    * This test will be skipped if the image tag is the latest WebLogic image tag
```

Comment:
We need not skip the test if the tag is latest i.e. 14.1.1.0-x, since we are tagging the same image to an image with a time stamp appended.  Modify the description.

Reply:
We won't skip the test if the tag being input from Jenkins run is 14.1.1.0-x since 
String imageUpdate = KIND_REPO != null ? KIND_REPO
        + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
        : WEBLOGIC_IMAGE_NAME + ":" + imageTag;

So it all depends on WEBLOGIC_IMAGE_NAME.
I'm going to modify the comment though.

Reply:
Please run it against the OKE cluster as well.

Reply:
Sure. Thanks!

Reply:
@maggiehe00 please coordinate with Marina to run the tests on OKE.

Reply:
I have already kicked off a run as @sankarpn suggested
"Please run this IT class against the OKE cluster in job https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/".

Reply:
Just aborted it after talking to Marina.

Reply:
OKE test passed at
https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/338/testReport/